deploy/operator/ztp/assisted-installer-crds-playbook.yaml

```diff
@@ -25,13 +25,14 @@
     - cluster_host_prefix_additional: "{{ lookup('env', 'CLUSTER_HOST_PREFIX_ADDITIONAL') }}"
     - external_subnet_additional: "{{ lookup('env', 'EXTERNAL_SUBNET_ADDITIONAL') }}"
     - service_subnet_additional: "{{ lookup('env', 'SERVICE_SUBNET_ADDITIONAL_ADDITIONAL') }}"
-    - extra_baremetalhosts: "{{ lookup('file', lookup('env', 'EXTRA_BAREMETALHOSTS_FILE')) | from_json }}"
+    - baremetalhosts: "{{ lookup('file', lookup('env', 'EXTRA_BAREMETALHOSTS_FILE')) | from_json }}"
```

Comment:
`EXTRA_BAREMETALHOSTS_FILE` contains the definition of all the extra nodes created by dev-script, which are currently used to deploy define the `master` nodes. 

I'm saying this because I think we should find a way to differentiate between `masters` and `workers` here. Here's an idea:

Have 2 variables `SPOKE_NUM_MASTERS` and `SPOKE_NUM_WORKERS`. Master nodes will be the first `$NUM_MASTERS` hosts defined in the json and workers will use the rest of the hosts. thoughts?

Reply:
I like this idea but unsure if we actually need it for RWN. Though we should look into defining the `SPOKE_NUM_MASTERS` and `SPOKE_NUM_WORKERS` to clarify what is going on here. 🤔 

I think also with Pawan's flow he initally wanted to use different VMs for the different SNO clusters. He wanted to put them in separate manifest files as well. With that said perhaps I should think about how we can interact with this a bit more hm.

The flow I'm thinking is:
1) Setup hub cluster w/ dev-scripts, define `EXTRA_BAREMETAL_HOSTS` and use this for the spoke masters.
2) Install assisted operator and set up spoke cluster
3) Run `remote_nodes.sh` in dev-scripts, this will define `REMOTE_BAREMETAL_HOSTS` and we can use this for the spoke workers.
4) Finally we should be able to apply the remote nodes to the spoke namespace and assisted should it pick it up from there? I think.. 😅

Reply:
Sounds reasonable to me! 

While reviewing this PR yesterday, I forgot that `dev-scripts` creates a separate hosts file when calling `remote_nodes.sh`. Now that I remembered this, the changes in this PR make a lot more sense :) 

Let me know if I can help with tests or something :)